### DISCUSSION TITLE
Is exporter keeps what data was scraped?

### DISCUSSION THREAD

**Discussion by max-teren** [mvisonneau/gitlab-ci-pipelines-exporter#563] [General]
If I deployed gitlab-ci-pipelines-exporter with disabled Redis, does it keep the state of what pipelines and jobs were scrapped, or after each restart it starts to fetch all data from the beginning?

**Comment by f0rkth1s** [comment]
I need to understand this also, every time I restart the exporter or Prometheus, the data scrape starts fresh. I have the Prometheus data stored on efs so it will not be lost.
